src/Microsoft.Health.Dicom.Functions/host.json

```diff
@@ -8,10 +8,21 @@
       }
     }
   },
-  "indexing": {
-    "add": {
-      "batchSize": 1000,
-      "parallelBatches": 10
+  "DicomFunctions": {
```

Comment:
This project/runtime needs to be a dummy worker. All feature settings need to happen in one place in Dicom web app.

Reply:
How we could do that? I mean one is host.json, the other is appsetting.json.

Reply:
Talked with Will, there will be another PR for moving AddTags logic back to DicomServer, in this way, all feature configs should be removed from here.
For this PR, I just remove them from `host.json`